### ISSUE TITLE
Add options to the CLI tool

### ISSUE THREAD

**Issue by Alan-Cha** [strongloop/oasgraph#44]
Currently, the CLI tool just starts up a GraphQL interface. However, with recent developments thanks to people like @marioestradarosa, we are making the CLI tool much more sophisticated. Following this trend, the next step would be to add [options](https://github.com/strongloop/oasgraph#options) to the CLI tool. 

Perhaps we will not be able to add all of the options but at least for a few of them, such as the boolean ones, they should be easy to add.

**Comment by marioestradarosa** [comment]
I propose to instead of options, it should support a config file, to be referenced as follows:

`oasgraph oasapi.json --config myconfig.json` 

and inside that file you should place all configurations.

**Comment by Alan-Cha** [comment]
Yeah, I think that makes sense. Some of the options are quite complicated and would not be easily representable as CLI arguments.

**Comment by Alan-Cha** [comment]
Then again, do we really want to give the CLI tool all the features of the library function at the cost of ease of use? 

I think we need to figure out exactly what we want the CLI tool to be used for. 

If the purpose of the CLI tool is just for demo and testing purposes, then perhaps just added some of the boolean options as CLI arguments will be more than enough. 

If the purpose of the CLI tool is to be an "alternative" to the library function, which is already sketchy to say because we list the CLI tool `dependencies` as `devdependencies`, then we will like need to bring out the big guns and support a config file. 

We could always support both but again, we should try to figure out of goals for the CLI tool.

**Comment by marioestradarosa** [comment]
I think the CLI tool souldn't be too powerful in features etc. The reason why is because it will just needed  to quickly test or un a POC against an OPEN API  REST application. I will never used this cli tool in production. 

So in my opinion, the CLI should deliver:

- A way to specify local or remote OAS file
- A way to save the generate the file locally to edit it in a separate editor
- A way to specify its port number because I might be running others too

Aside from that, I as a user won't need any other feature for now, since the main purpose is to incorporate OASGraph library into my application.
